example-environment.lua

```diff
@@ -411,7 +427,7 @@ end
 -- MQTT subscribing: We use this to open the network for joining
 M:subscribe("/zigbee-lua/permit_join")
 ctx.task{name="mqtt_permit_join",function()
-  for ok, msg in ctx:wait_all({"mqtt_client", "message"}, function(msg) return msg.topic == "/zigbee-lua/permit_join" end) do
+  for _ in ctx:wait_all({"mqtt_client", "message"}, function(msg) return msg.topic == "/zigbee-lua/permit_join" end) do
     U.DEBUG("mqtt_environment", "got permit_join message via MQTT, opening network for devices")
```

Comment:
for documentation/example purposes, I think it would be good to keep these unused ones.

Reply:
Ok,  ignore W213 (Unused loop variable) in .luacheckrc globally.

Reply:
Hmmmmm, yes, probably. At least in the example stuff...